frontend/src/components/user-details/components/user-profile/components/avatar-wrapper/avatar-wrapper.tsx

```diff
@@ -23,6 +70,7 @@ const AvatarWrapper: FC = () => {
           label="Update File"
           btnType="upload"
           className={styles.marginBottom}
+          onClick={handleOpenFileBrowser}
```

Comment:
```suggestion
```
you need to render input with type `file` if we provide a `btnType="upload"` prop

Reply:
@what1s1ove In that case, I guess, we don't need to open file browser on click to avatar, do we?

Reply:
right. only by clicking on the button